pkg/descheduler/strategies/topologyspreadconstraint_test.go

```diff
@@ -3,9 +3,8 @@ package strategies
 import (
 	"context"
 	"fmt"
-	"testing"
-
 	"sigs.k8s.io/descheduler/pkg/api"
+	"testing"
```

Comment:
May I know the reason why moving this from L6 to L7?

Reply:
it was auto format in the IDE, reverted the change.

Reply:
``
`
./hack/verify-govet.sh
./hack/verify-spelling.sh
./hack/verify-gofmt.sh
!!! 'gofmt -s' needs to be run on the following files: 
./pkg/descheduler/strategies/topologyspreadconstraint_test.go
make: *** [Makefile:107: verify-gofmt] Error 1
```